buildenv/jenkins/jobs/builds/Build-linux_riscv64_cross

```diff
@@ -0,0 +1,74 @@
+def setBuildStatus(String message, String state, String sha) {
+    context = "continuous-integration/eclipse-omr/branch/linux_riscv64"
+    step([
+        $class: "GitHubCommitStatusSetter",
+        reposSource: [$class: "ManuallyEnteredRepositorySource", url: "https://github.com/eclipse/omr"],
+        contextSource: [$class: "ManuallyEnteredCommitContextSource", context: context],
+        errorHandlers: [[$class: "ChangingBuildStatusErrorHandler", result: "UNSTABLE"]],
+        commitShaSource: [$class: "ManuallyEnteredShaSource", sha: sha ],
+        statusBackrefSource: [$class: "ManuallyEnteredBackrefSource", backref: "${BUILD_URL}flowGraphTable/"],
+        statusResultSource: [$class: "ConditionalStatusResultSource", results: [[$class: "AnyBuildResult", message: message, state: state]] ]
+    ]);
+}
+
+pipeline {
+    agent{label 'linux && x86 && compile:riscv64'}
+    environment {
+        PATH = "/usr/lib/ccache/:$PATH"
+        GTEST_COLOR = "0"
+    }
+    stages {
+        stage('Get Sources') {
+            steps {
+                timestamps {
+                    checkout scm
+                    setBuildStatus("In Progress","PENDING","${env.GIT_COMMIT}")
+                }
+            }
+        }
+        stage('Build') {
+            steps {
+                timestamps {
+                    echo 'Output CCACHE stats before running and clear them'
+                    sh '''ccache -s -z'''
+
+                    dir('build_native') {
+                        echo 'Configure...'
+                        sh '''cmake ..'''
+
+                        echo 'Compile...'
+                        sh '''make -j4'''
+                    }
+
+                    dir('build') {
+                        echo 'Configure...'
+                        sh """cmake -Wdev -DOMR_COMPILER=ON -DOMR_TEST_COMPILER=ON -DOMR_JITBUILDER=ON -DOMR_JITBUILDER_TEST=ON \
```

Comment:
You shouldn't need to enable these feature explicitly, they're enabled (either directly or indirectly) by `caches/Travis.cmake`.